### ISSUE TITLE
Loss not decrease when training

### ISSUE THREAD

**Issue by ethanyhzhang** [Richardzhangxx/AMR-Benchmark#2]
As I train with the source code, ensuring the version of tf and Keras are the same, the training loss almost remain the same all the time until finish. At last the acc is very bad. 

Have you encountered the same issue?

**Comment by yanfengneng** [comment]
我按照作者本人的 tf 和 keras 版本一直跑不成功，我不知道问题出在哪里了，有人能提供一些帮助吗

**Comment by Faith594** [comment]
@ethanyhzhang You need to change the version of Tensorflow from 1.14 to 2.7
